Tribler/Test/Core/test_encoding.py

```diff
@@ -0,0 +1,192 @@
+from nose.tools import raises
+from Tribler.Core.Utilities.encoding import _a_encode_int, _a_encode_long, _a_encode_float, _a_encode_unicode, \
+    _a_encode_bytes, _a_encode_list, _a_decode_true, _a_decode_false, _a_decode_none, _a_encode_set, _a_encode_none, \
+    _a_encode_bool, _a_encode_tuple, _a_encode_dictionary, bytes_to_uint, _a_decode_int, _a_decode_long, _a_decode_float, \
+    _a_decode_unicode, encode, _a_decode_bytes, _a_decode_list, _a_decode_mapping, _a_decode_set, _a_decode_tuple, \
+    _a_decode_dictionary, decode
```

Comment:
Sorry I didn't see that one before, but we use parens-wrapped multiline imports in Tribler.